test/converters/junitxml/junitxml.go

```diff
@@ -0,0 +1,31 @@
+package junitxml
+
+import (
+	"strings"
+
+	"github.com/bitrise-io/go-utils/fileutil"
+)
+
+type Handler struct {
+	files       []string
+	xmlFilePath string
+}
+
+func (h *Handler) SetFiles(files []string) {
+	h.files = files
+}
+
+func (h *Handler) Detect() bool {
+	// TODO: detect more and validate if Junit4 xml
+	for _, file := range h.files {
+		if strings.HasSuffix(file, ".xml") {
+			h.xmlFilePath = file
+			return true
+		}
+	}
+	return false
+}
+
+func (h *Handler) XML() ([]byte, error) {
+	return fileutil.ReadBytesFromFile(h.xmlFilePath)
```

Comment:
parse it to our shared model, to filter out unknown/unused fields of the resports

Reply:
same as https://github.com/bitrise-io/steps-deploy-to-bitrise-io/pull/79#discussion_r257692466